### PR TITLE
Fix cargo build on Linux

### PR DESCRIPTION
**Pull request by mikemadden42**
These updates fix the following build issue on Linux.

```
error: failed to run custom build command for `openssl v0.9.24`

Caused by:
  process didn't exit successfully: `/home/michael_madden/rust/xori/target/release/build/openssl-62f9983eb3104bc7/build-script-build` (exit code: 101)
--- stderr
thread 'main' panicked at 'Unable to detect OpenSSL version', /home/michael_madden/.cargo/registry/src/github.com-1ecc6299db9ec823/openssl-0.9.24/build.rs:16:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

There were also quite a few whitespace issues in `src/bin/pesymbols.rs`.

### CODE REVIEW

**Comment by mikemadden42** [comment]
This PR address issue https://github.com/endgameinc/xori/issues/40.

File: src/bin/pesymbols.rs [hunk 1]
```diff
@@ -51,10 +51,10 @@ fn download_pdb(
     println!("{}\n{}", url, user_agent);
     let mut response = Client::new()
             .get(url)
-            .header(UserAgent::new(user_agent.clone()))
+            .header(USER_AGENT, "curl/7.65.3")
```
Comment (iximeow) [inline src/bin/pesymbols.rs]: assuming you're passing a `&'static str` here to satisfy the type system more than to disregard the original `user_agent`, you could convert the provided `&String` to a `&str` by `&*user_agent`. That said `&String` almost always should be `&str` for API friendliness reasons.

Comment (mikemadden42) [reply src/bin/pesymbols.rs]: Very solid point @iximeow .  I am using the `user_agent` in the last commit.  How does this look?

Comment (iximeow) [reply src/bin/pesymbols.rs]: looks good! :+1:

**Comment by rseymour** [comment]
LGTM

**Pull request merged**